javascript/ql/lib/semmle/javascript/frameworks/SQL.qll

```diff
@@ -368,10 +368,10 @@ private module SpannerCsv {
       // package; type; path; kind
```

Comment:
This `// package; type; path; kind` comment should be updated (and similar comments, if there are any).